tests/unit/reorder-timeline.test.ts

```diff
@@ -2,22 +2,28 @@ import type { mastodon } from 'masto'
 import { describe, expect, it } from 'vitest'
 
```

Comment:
we might need to run this within a happy-dom or nuxt environment if it's trying to access window object